### PR TITLE
Test PR

### PR DESCRIPTION
**Pull request by garethsb**
Look here.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ This repository holds the source for this Specification, part of the family of [
 
 - It helps ensure consistency between NMOS Specifications.
 - It helps us test our continuous integration.
+- It just does
```
Comment (cristian-recoseanu) [inline README.md:18]:
```suggestion
- It just does
```
Does it?

Comment (pedro-alves-ferreira) [reply README.md:18]: Me too

Comment (peterbrightwell) [reply README.md:18]: LGTM too

Comment (pedro-alves-ferreira) [reply README.md:18]: My review

File: README.md [hunk 2]
```diff
@@ -15,6 +15,7 @@ This repository holds the source for this Specification, part of the family of [
 
 - It helps ensure consistency between NMOS Specifications.
```
Comment (cristian-recoseanu) [inline README.md:16]:
```suggestion
- It helps ensure consistency between NMOS Specifications.
```
It does indeed.

**Review by garethsb — COMMENTED** [review]
I don't like it

**Comment by peterbrightwell** [comment]
Well we don't really want it merged so closing :)

**Pull request closed without merging**